src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -55,20 +55,20 @@
         <div class="row">
           <div class="col-12">
             <div class="row padding-15px text-left">
-              <div class="col-md-2">
+              <div class="col-lg-3">
```

Comment:
I am unsure about changing this for your solution.

As I understand it, your change from md (medium) to lg (large) makes it so that the table collapses into panel form sooner (in this case, as width goes below 768px instead of 480px).

However, this potentially breaks the normal page and causes inconsistency among other pages that may be still using md, and we're adapting the page specifically to fit the issue in the help section, which is less useful than ensuring the page works in normal usage.

If we go down this path of fixing each individual section that breaks on scrolling, many more may need to be fixed.
<img width="572" alt="Screenshot 2021-02-08 at 10 03 04 AM" src="https://user-images.githubusercontent.com/4904988/107168035-7ec9f200-69f5-11eb-980d-ee58096f0afc.png">

Reply:
I think it is a great consideration. Thanks for pointing out!

Removed the changes in `question-edit-form` and applied the similar idea only in `instructor-help-page`.

Also updated the latest snapshot in PR description.